packages/twenty-front/src/modules/object-record/utils/generateEmptyFieldValue.ts

```diff
@@ -49,8 +49,21 @@ export const generateEmptyFieldValue = (
       };
     }
 
-    case FieldMetadataType.MultiSelect:
-    case FieldMetadataType.Select: {
+   // Handling MultiSelect and Select in the default case
+    default: {
+      if (
+        fieldMetadataItem.type === FieldMetadataType.MultiSelect ||
```

Comment:
I am not sure to understand why you choose to handle this in the default section. This should stay in their own case

Reply:
Also multi select is not implemented yet on the backend so it should stay has non implemented